content/acra/configuring-maintaining/general-configuration/acra-server.md

```diff
@@ -211,9 +211,13 @@ weight: 3
 
 * `--keystore_cache_size=<count>`
 
-  Count of keys that will be stored in in-memory LRU cache in encrypted form.
-  Use `0` to set unlimited size, `-1` to disable caching.
-  Default is `0`.
+  Maximum number of keys stored in in-memory LRU cache in encrypted form. 0 - no limits, -1 - turn off cache
+  Default is `1000`.
```

Comment:
imho, we can mention that it is since 0.92.0. at least for ourselves, it will be useful in the future.